.github/workflows/sync-onnxrt-main.yaml

```diff
@@ -1,46 +1,61 @@
 name: Onnxruntime main weekly sync
 on:
   schedule:
-    - cron: "05 17 * * 1"
+    - cron: '27 1 * * 4'
 
 jobs:
-  runs-on: ubuntu-latest
-  sync:
+  createPullRequest:
+    name: Update and create pull request
+    runs-on: self-hosted
     steps:
-      - uses: actions/checkout@v3
-        with:
-          ref: develop
-          path: ../
+      - name: cleanup-old-runs
+        run: rm -rf onnxruntime
```

Comment:
I think there is a better way without having to rm and clone.  

git ls-remote https://github.com/microsoft/onnxruntime.git HEAD | awk '{ print $1}'

That will print just the sha value so you can direct that to where you need it

Reply:
Perfect. Done